hubblestack_nova/cve_scan.py

```diff
@@ -0,0 +1,43 @@
+# -*- encoding: utf-8 -*-
+'''
+Hubble Nova plugin for FreeBSD pkgng audit
+
+:maintainer: HubbleStack
+:maturity: 20160428
+:platform: Unix
+:requires: SaltStack + oscap module
+'''
+from __future__ import absolute_import
+import salt.utils
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    if not (utils.is_linux() and utils.which('oscap')):
+        return False, 'This module requires Linux and the oscap binary'
+    return True
+
+
+def audit(data_list, tags, verbose=False):
+    '''
+    Run the network.netstat command
+    '''
+    ret = {'Success': [], 'Failure': []}
+
+    __tags__ = []
+    __feed__ = ''
+    for data in data_list:
+        if 'cve_scan' in data:
+            __tags__ = ['cve_scan']
+            for k, v in data.items():
+                __feed__ = v
```

Comment:
This is not quite right. Theoretically someone could put `pkg` data in the same yaml file as `cve_scan` data, so you need to specifically pull out the `cve_scan` key:

```
    for data in data_list:
        if 'cve_scan' in data:
            __tags__ = ['cve_scan']
            __feed__ = data['cve_scan']
```